### ISSUE TITLE
REQUEST: Different keyboard sortcuts sschemes for tiling window manage and floating window manager

### ISSUE THREAD

**Issue by aimforthebushes** [pop-os/pop#1020]
I think there sould be two different sets of keyboard shortcuts. 

One for the tiling window manager and one for the regular window manager. That way you could use a keyboard shortcut to do one action on regular window management and a different action when on tiling mode.

**Comment by bublik77** [comment]
I agree with @aimforthebushes  100 percent. I using pop_os since version 18.10, and now very hard to get used for new keyboard shortcuts when you are not using tiling mode

**Comment by WatchMkr** [comment]
Which shortcuts are giving you trouble?

**Comment by bublik77** [comment]
for example to place program like in screenshot I use Ctrl+Alt + -> or <- , now it not work 
![Screenshot from 2020-05-13 17-53-00](https://user-images.githubusercontent.com/16838069/81829482-9ac26500-9543-11ea-9cad-8e110587bdcc.png)

**Comment by luispjoaquim** [comment]
> for example to place program like in screenshot I use Ctrl+Alt + -> or <- , now it not work
> ![Screenshot from 2020-05-13 17-53-00](https://user-images.githubusercontent.com/16838069/81829482-9ac26500-9543-11ea-9cad-8e110587bdcc.png)

This might be related with this #1022

**Comment by bublik77** [comment]
> Which shortcuts are giving you trouble?

I wound not be said that is trouble, it is more like an inconvenience when you habit to something  =)

**Comment by bublik77** [comment]
> This might be related with this #1022

It looks like yes, sorry for clogs the branch

**Comment by WatchMkr** [comment]
That is a bug. A fix is coming soon.
